packages/lodestar-core/src/types/block.ts

```diff
@@ -0,0 +1,30 @@
+import {Lightclient, SignedBeaconBlock, BeaconBlock, Slot} from "@chainsafe/lodestar-types";
+import {ContainerType} from "@chainsafe/ssz";
+import {IBeaconConfig} from "@chainsafe/lodestar-config";
+import {EMPTY_SIGNATURE} from "@chainsafe/lodestar-beacon-state-transition";
+
+export type SignedBeaconBlockType = SignedBeaconBlock | Lightclient.SignedBeaconBlock;
+export type BeaconBlockType = BeaconBlock | Lightclient.BeaconBlock;
+
+export function getSignedBeaconBlockSSZTypeBySlot(
```

Comment:
I thought we were going to go with:
`config.getTypes(epoch/slot).SignedBeaconBlock`

In that case we shouldn't need this `lodestar-core` library, at least not right now. Preference for not having it to make merging this branch easier when we need to.